scripts/test-run.sh

```diff
@@ -27,20 +27,51 @@ TEST_ENV_NAME="${TEST_ENV_NAME:-kong-charts-tests}"
 KUBECTL="kubectl --context kind-${TEST_ENV_NAME}"
 KUBERNETES_VERSION="$($KUBECTL version -o json | jq -r '.serverVersion.gitVersion')"
 
+# ------------------------------------------------------------------------------
+# Deploy Kuma configuration and test namespace
+# ------------------------------------------------------------------------------
+
+echo "---
+apiVersion: kuma.io/v1alpha1
+kind: Mesh
+metadata:
+  name: default
+spec:
+  mtls:
+    backends:
+    - conf:
+        caCert:
+          RSAbits: 2048
+          expiration: 10y
+      dpCert:
+        rotation:
+          expiration: 1d
+      name: ca-1
+      type: builtin
+    enabledBackend: ca-1
+---
+apiVersion: v1
+kind: Namespace
+metadata:
+  labels:
+    kuma.io/sidecar-injection: enabled
+  name: ${RELEASE_NAMESPACE}
+" | kubectl --context "kind-${TEST_ENV_NAME}" apply -f -
+
 # ------------------------------------------------------------------------------
 # Deploy Chart - Kubernetes Ingress Controller
 # ------------------------------------------------------------------------------
 
 if [[ "${TAG}" == "default" ]]
 then
     echo "INFO: installing chart as release ${RELEASE_NAME} to namespace ${RELEASE_NAMESPACE}"
-    helm install --create-namespace --namespace "${RELEASE_NAMESPACE}" "${RELEASE_NAME}" \
```

Comment:
Why remove this 🤔?

Reply:
We're now creating the namespace along with the Mesh resource above. We can't set `kuma.io/sidecar-injection: enabled` on it with Helm and want that annotation in place before Kong's Deployment is created, since Kuma won't update existing resources without a restart.